lib/node_modules/@stdlib/math/base/special/cphase/README.md

```diff
@@ -38,12 +38,14 @@ The [argument][complex-number-argument] of a complex number, also known as the *
 var cphase = require( '@stdlib/math/base/special/cphase' );
 ```
 
-#### cphase( re, im )
+#### cphase( z )
 
-Computes the [argument][complex-number-argument] of a `complex` number comprised of a **real** component `re` and an **imaginary** component `im`.
+Computes the [argument][complex-number-argument] of a double-precision complex floating-point number.
```

Comment:
Can you also update the main README description to use the `double-precision` language.